src/gfxbase.cpp

```diff
@@ -60,15 +60,15 @@
     template<>
     void GFXBase::FillGetNoise<NoiseApproach::Two>()
```

Comment:
This mismatch of WIDTH and MATRIX_WIDTH (etc) causes a crash in JSONSerializer!  Heap corruption, I assume...

Reply:
Where exactly? "JSONSerializer" is not an actual thing at the codebase level - it's the name of a .h/.cpp file pair that contain JSON (de)serialization scaffolding. 

I'm guessing you're actually referring to some class' `IJSONSerializable::SerializeToJSON()` or `IJSONSerializable::DeserializeFromJSON()` implementation?